addons/point_of_sale/static/src/js/printers.js

```diff
@@ -30,7 +30,7 @@ class PrintResultGenerator {
             successful: false,
             message: {
                 title: _t('Connection to the printer failed'),
-                body: _t('Please check if the printer is still connected. \n' +
+                body: _t('Please check if the printer is still connected and make sure there are no errors in the IoT box logs. \n' +
```

Comment:
Once again, the intention is nice but it can be misleading as well.
In the sense that if we assumed that they have a ton of unrelated errors in their log files (for example because the initial setup was done badly), they might be confused and try to solve every errors in their logs which are not related to the issue.
Maybe we should replace it with something like: `your IoT may have logged more information regarding printing errors` (intentionally use words that you set in the logger error).